Firebase/Auth/Source/FIRAuth_Internal.h

```diff
@@ -124,6 +150,13 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (BOOL)signOutByForceWithUserID:(NSString *)userID error:(NSError *_Nullable *_Nullable)error;
 
+/** @fn operationType
+    @param operationType The value of the FIRAuthOperationType enum which will be translated to its
+        corresponding string value.
+    @return The string value corresponding to the FIRAuthOperationType argument.
+ */
+NSString *const operationType(FIRAuthOperationType operationType);
```

Comment:
Please prefix the global function, e.g. `FIRAuthStringWithOperationType`, or`FIRAuthOperationTypeString`, or even `FIRAuthOperationString`.

Reply:
Done.